### PR TITLE
[RHDM-349]: Added drools playground dependency

### PR DESCRIPTION
**Pull request by adrielparedes**
@porcelli could you review?

https://issues.jboss.org/browse/RHDM-349

Related to:
https://github.com/kiegroup/optaplanner-wb/pull/250
https://github.com/kiegroup/drools-wb/pull/799
https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/692
https://github.com/kiegroup/kie-wb-distributions/pull/685
https://github.com/kiegroup/kie-wb-playground/pull/46
https://github.com/kiegroup/jbpm-wb/pull/980
https://github.com/kiegroup/kie-wb-common/pull/1411

### CODE REVIEW

File: optaplanner-bom/pom.xml [hunk 1]
```diff
@@ -418,6 +418,12 @@
         <version>${version.org.kie}</version>
         <classifier>sources</classifier>
       </dependency>
+      <!--KIE Drools Workbench Playground -->
+      <dependency>
+        <groupId>org.kie.workbench.playground</groupId>
+        <artifactId>drools-playground-parent</artifactId>
```
Comment (ge0ffrey) [inline optaplanner-bom/pom.xml]: This will break the build because this new repository isn't added to the repository-list.txt

**Comment by ge0ffrey** [comment]
Yet another repository? We already have kie-wb-playground. Please just make it a module in there.
Every git repository increases the build-all time seriously and increases the chance of jenkins timing out and reporting a false negative.

Furthermore, the name of the repository is invalid because it contains both the drools and the optaplanner examples.

**Comment by adrielparedes** [comment]
@ge0ffrey @jhrcek ok, I think I have a better solution, let me test it and upload it

**Comment by adrielparedes** [comment]
@ge0ffrey @jhrcek I've made some changes, could you review them please?

**Pull request merged**